docs/getting-started/key-concepts/looping.md

```diff
@@ -81,6 +81,7 @@ There are a limited number of nodes and operations where you need to design a lo
 	* **List**: This operation executes only once, not for each incoming item.
 * [**Google Sheets**](../../nodes/nodes-library/nodes/GoogleSheets/README.md) node:
 	* **Read**: This operation will execute only once for the `Sheet` resource.
+	* **Update**: This operation will update multiple rows only if they are in the same range.
```

Comment:
Could we say something like:

"This operation updates multiple rows if they are in the same range, or [whatever it does if they are not - updates a single row I guess?]"

Reply:
Thx for your feedback on this one @StarfallProjects, clarified what's happening in https://github.com/that-one-tom/n8n-docs/commit/7efac0f80933254cd4b7417d18e79baa5332163f.